src/modules/users/dto/user_docs.dto.ts

```
@@ -84,4 +97,33 @@ export class CreateUserDocDTO {
   doc_datatype: string;
+
+  @ApiProperty({
+    description: 'Whether the watcher is registered',
+    example: false,
+    required: false,
+  })
+  @IsOptional()
+  @IsBoolean()
+  watcher_registered?: boolean;
+
+  @ApiProperty({
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)